vault/dendron.tutorial.linking-notes.md

```diff
@@ -2,43 +2,59 @@
 id: 9MZBqhrijEM4QpZRa5t08
 title: Linking Notes
 desc: Linking Notes
-updated: 1637551418396
+updated: 1639532701850
 created: 1625563999532
 nav_order: 2
 ---
 
 ### Links
 
-Dendron supports links between your notes, which can help you organically connect your thoughts and build up a powerful and **personal** knowledge graph.
+Dendron supports links between your notes, which can help you organically connect your thoughts and build up a powerful and personal knowledge graph.
```

Comment:
"powerful and personal knowledge graph" is a mouthful.

Would use the following:
> Dendron helps you create links between notes. These links help you organically discover relationships and build out your knowledge graph.

Principles:
- avoid fluff: less adjectives and get to the meat
- for the tutorial, we don't want to restrict to personal knowledge management since dendron can be used for more than that